CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing Guidelines
+This repo contains an in-progress prototype of some of the foundational aspects of V2 of Kubernetes Federation. All the contribution is done via GitHub. Please also read the contributors guide: https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
```

Comment:
Replace in-progress prototype to say alpha release since we've officially released our first alpha.